app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd_list = command.split()
+
+    if (
```

Comment:
add `else` with message that user should write correct data

Reply:
Done.